Modules/eventp/locale/fr_FR/LC_MESSAGES/eventp_messages.po

```diff
@@ -82,7 +82,7 @@ msgid "Send Email"
 msgstr "Send Email"
 
 #: ../../../eventp_processlist.php:51
-msgid "Send an email to the user with the specified body."
+msgid "Send an email to the user with the specified body. Email sent to user's email address or default set in config."
```

Comment:
hi @thib66 
I have had to change the English string that you translated to better explain how emails are sent. I thought I'd let you know in case you might want to edit the french version?
this is what google translate suggested...
> Send an email to the user with the specified body. **Email sent to user's email address or default set in config.**
> Envoyer un e-mail à l'utilisateur avec le corps spécifié. **Email envoyé à l'adresse e-mail de l'utilisateur ou par défaut défini dans config.**

Reply:
Hello, a better translation would be : 
Email envoyé à l'adresse email de l'utilisateur ou à celle définie par défaut dans la configuration.
Regards,
Aymeric